Sources/ScoutCLT/Read/ReadCommand.swift

```diff
@@ -57,72 +57,24 @@ struct ReadCommand: ParsableCommand, ExportCommand {
 
     // MARK: - Functions
 
-    func run() throws {
-        let data = try readDataOrInputStream(from: inputFilePath)
-        try read(from: data)
-    }
-
-    func read(from data: Data) throws {
-
+    func inferred<P: PathExplorer>(pathExplorer: P) throws {
         let readingPath = self.readingPath ?? Path()
+        var explorer = try pathExplorer.get(readingPath)
+        let value = try getValue(from: &explorer)
+        let colorInjector = try self.colorInjector(for: exportFormat ?? explorer.format)
 
-        do {
-            let (value, injector) = try readValue(at: readingPath, in: data)
-
-            if value == "" {
-                throw RuntimeError.noValueAt(path: readingPath.description)
-            }
-
-            if let output = outputFilePath?.replacingTilde, let contents = value.data(using: .utf8) {
-                let fm = FileManager.default
-                fm.createFile(atPath: output, contents: contents, attributes: nil)
-                return
-            }
-
-            let output = colorise ? injector.inject(in: value) : value
-
-            print(output)
+        if value == "" {
```

Comment:
For a `String`, it would be better to use `.isEmpty` to check if the content is empty.

Reply:
Right'o!

Reply:
Fixed in latest commit 739ca313022c667ae53fc96f79a81c50d6a8264f